fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1278,6 +1278,24 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static boolean enable_statistic_collect = true;
 
+    /**
+     * Enable full statistics collection
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_collect_full_statistics = false;
```

Comment:
All statistics configs would better name with the same prefix.

Reply:
Numerical configuration, I have changed to statistic prefix, enable/disable I keep the beginning of enable like other codes